scripts/popup.js

```diff
@@ -353,15 +381,19 @@
     }
 
     function startDownload() {
-      var checkedImages = 0;
+      var checkedImages = [];
       for (var i = 0; i < visibleImages.length; i++) {
         if ($('#image' + i).hasClass('checked')) {
-          checkedImages++;
-          chrome.downloads.download({ url: visibleImages[i] });
+          checkedImages.push(visibleImages[i]);
         }
       }
+      ls.image_count = checkedImages.length;
+      ls.image_number = 1;
+      checkedImages.forEach(function(checkedImage) {
+        chrome.downloads.download({ url: checkedImage });
+      });
 
-      flashDownloadingNotification(checkedImages);
+      flashDownloadingNotification(ls.image_count);
```

Comment:
I opted for passing the variable here to keep the `flashDownloadingNotification` function more pure and avoid typos like `ls.image_count` vs `ls.imageCount`. I know it already uses `ls` for some other values, but I think the less it uses global state, the better.